src/js/modules/articles/components/SearchForm.js

```diff
@@ -62,28 +62,42 @@ const styles = {
   },
 };
 
-const SearchForm = ({ classes, handleSubmit, submitting }) => {
-  return (
-    <form onSubmit={handleSubmit} className={classes.SearchForm}>
-      <div className={classes.inputContainer}>
-        <Search className={classes.search} />
-        <Field
-          name="search"
-          type="text"
-          component="input"
-          placeholder="Enter search terms"
-        />
-        <button
-          type="submit"
-          disabled={submitting}
-          className={classes.submitButton}
-        >
-          Go
-        </button>
-      </div>
-    </form>
-  );
-};
+class SearchForm extends PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  componentWillMount() {
+    const { query, initialize } = this.props;
+    if (query) {
+      initialize({ query });
```

Comment:
Why are you storing query in the local state, then adding it to reduxForm? Seems a little redundant. Just pull it from the route, then initialize the form with it

Reply:
Pretty sure I have CMS code that's similar